ApollosProject/apollos-prototype
# Fixes text theme colors in navigation headers

## DESCRIPTION

### What does this PR do, or why is it needed?
This uses the `action.primary` color for navigation "back" buttons and `text.primary` for navigation titles.

### What design trade-offs/decisions were made?
N/A

### How do I test this PR?
Open the app tap on `Discover` > `View All` on one of the content channels. The navigation header should have the correct colors.

You can also check `Connect` > `Settings`. The navigation header should have the correct colors.

### What automated tests did you write?
Unfortunately, there are no snapshots tests over complete screens at the moment.

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
